utils.py

```diff
@@ -42,6 +42,78 @@ def poly2mask(vertex_row_coords, vertex_col_coords, shape):
     return mask
 
 
+def get_points(data, img_path, roi_points, shape):
```

Comment:
This naming is totally workable...and for the urgency of the matter is "a pass".

However, it does not convey the "action" it is performing.  Not that I have the best drop-in name replacement ready.

*fill_data_with_labeled_roi*. Conveys... but doesn't have that "snap" that the simple "get_points" does.

*label_to_voxels* ?